covid19-etl/etl/ds4c.py

```diff
@@ -78,12 +65,16 @@ def files_to_submissions(self):
                     subject["date_death_or_discharge"] = deceased_date
 
                 state = row[header["state"]]
-                if state == "isolated":
-                    subject["isolated"] = "True"
-                elif state == "released":
-                    subject["recovered"] = "recovered"
-                elif state == "deceased":
-                    subject["death"] = "True"
+                if state:
+                    subject["current_state"] = state
+
+                # explicitly remove old variables
+                subject["global_num"] = None
+                subject["infection_order"] = None
+                subject["disease"] = None
+                subject["isolated"] = None
+                subject["recovered"] = None
+                subject["death"] = None
```

Comment:
should `infected_by` be removed as well?

Reply:
PR is in Draft state and I'm waiting for Sean/LaRon input on proceeding with it: `infected_by` is an array now. :(